Disable tests relying on platform specific behaviour.

Known and tracked upstream:
https://github.com/espeak-ng/espeak-ng/issues/1368

--- a/tests/ssml.test
+++ b/tests/ssml.test
@@ -39,7 +39,7 @@
 for i in `ls tests/ssml/*.ssml` ; do test_ssml $i; done
 for i in `ls tests/ssml/*.ssml2` ; do test_ssml $i punct; done
 
-test_ssml_audio "<prosody>" fcb28f00670959980038171ceda3c6bdec61b884 "<speak><prosody rate=\"x-slow\" pitch=\"low\"> Slow and low </prosody><prosody rate=\"x-fast\" pitch=\"x-high\"> Fast and high.</prosody></speak>"
+#test_ssml_audio "<prosody>" fcb28f00670959980038171ceda3c6bdec61b884 "<speak><prosody rate=\"x-slow\" pitch=\"low\"> Slow and low </prosody><prosody rate=\"x-fast\" pitch=\"x-high\"> Fast and high.</prosody></speak>"
 # #410 is a bug in SSML. Sentence termination causes prosody stack to misfunction. 
 # Hash 00879af is the buggy version and should fail:
 test_ssml_audio "<prosody> bug #410" 00879aff11dc28a352736ca397ce9567e6f39029 "<speak><prosody rate=\"x-slow\" pitch=\"low\"> Slow and low. </prosody><prosody rate=\"x-fast\" pitch=\"x-high\"> Fast and high.</prosody></speak>"
